jobs/JWAVE_PREP

```diff
@@ -91,7 +91,12 @@ else
   if [ ! -L $ROTDIR/${WAVECUR_DID}.${RPDY} ]; then # Check if symlink already exists in ROTDIR
     $NLN $DMPDIR/${WAVECUR_DID}.${RPDY} $ROTDIR/${WAVECUR_DID}.${RPDY}
   fi
-  $NLN $DMPDIR/$CDUMP.${PDY}/$cyc/${WAVICEFILE} $ROTDIR/$CDUMP.${PDY}/$cyc/${WAVICEFILE}
+
+    ls -l $DMPDIR/$CDUMP.${PDY}/$cyc/${WAVICEFILE} 
+    ls -l $ROTDIR/$CDUMP.${PDY}/$cyc/${WAVICEFILE}
+  if [ ! -L $ROTDIR/${CDUMP}.${PDY}/${cyc}/${WAVICEFILE} ]; then # Check if symlink already exists in ROTDIR
```

Comment:
No sure why do you need this line.  NLN="ln -fs" will force a link

Reply:
I am not so sure why, but when I tried to run this on Hera the j-job exited when an ice file was found. In any case, it doesn't hurt to have this conditional block. Also, it is consistent with the current file block immediately above.

Reply:
I believe this was needed because when we run in free-forecast mode and our prep job hasn't run that ice file won't already be linked to from ROTDIR so we need to check for the symlink and create it if missing. Same with the rtofs symlink preceding this.